src/components/Catalog/CreateEventForm/CreateEventForm.jsx

```diff
@@ -111,6 +116,13 @@ const CreateEventForm = ({ eventData, setDataShouldRevalidate, closeModal }) =>
   const { filters, filterValues } = useSearchFilters();
   const [seasonFilter, yearFilter, subjectFilter, eventFilter] = filters;
 
+  useEffect(() => {
+    setValue('season', filterValues.season);
+    setValue('year', filterValues.year);
+    setValue('subject', filterValues.subject);
+    setValue('eventType', filterValues.eventType);
+  }, [filterValues]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'setValue'. Either include it or remove the dependency array.